Bottom: 207801d367791d6ffa8cb5d8a06d1049c2fb76aa
Top:    979925f0c38de7d565108736f8dfbd6307e6aa7b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 00:14:41 +0100

change.c: Fix some drawing artaefacts when rotating the buffer

I mixed up the calls to nofify_crosshair_changed() with true and
false parameters before and after the changes. This caused the GUI
to get out of sync undrawing and redrawing the crosshair attached
objects around the change.


---

diff --git a/src/action.c b/src/action.c
index ae6b87e..4f2e401 100644
--- a/src/action.c
+++ b/src/action.c
@@ -6040,7 +6040,7 @@ ActionPasteBuffer (int argc, char **argv, int x, int y)
   static char *default_file = NULL;
   int free_name = 0;
 
-  notify_crosshair_change (true);
+  notify_crosshair_change (false);
   if (function)
     {
       switch (GetFunctionID (function))
@@ -6147,7 +6147,7 @@ ActionPasteBuffer (int argc, char **argv, int x, int y)
 	      }
 	    else
 	      {
-		notify_crosshair_change (false);
+		notify_crosshair_change (true);
 		AFAIL (pastebuffer);
 	      }
